packages/create-flex-plugin/src/utils/validators.ts

```diff
@@ -111,7 +91,7 @@ const validate = async (config: FlexPluginArguments): Promise<FlexPluginArgument
         return process.exit(1);
     }
 
-    if (!_isSidValid('AC', config.accountSid)) {
+    if (!config.accountSid || !isSidOfType(config.accountSid, 'AC')) {
```

Comment:
we should check for null/undefined/empty string in `isSidOfType` so the callee doesn't have to worry about it

Reply:
or maybe in `isSidValid`?